frontend/src/features/properties/list/PropertyListView.tsx

```diff
@@ -0,0 +1,76 @@
+import './PropertyListView.scss';
+
+import React, { useEffect, useState, useMemo } from 'react';
+import queryString from 'query-string';
+import CustomAxios from 'customAxios';
+import { ENVIRONMENT } from 'constants/environment';
+import Table from 'components/Table';
+import { IPagedItems } from 'interfaces';
+import { IPropertyFilter, IProperty } from '.';
+import { columns as cols } from './columns';
+import { Container, Button } from 'react-bootstrap';
+
+const getPropertyListUrl = (filter: IPropertyFilter) =>
+  `${ENVIRONMENT.apiUrl}/properties/search/page?${filter ? queryString.stringify(filter) : ''}`;
+
+const filter: IPropertyFilter = {
+  page: 1,
+  quantity: 50,
+  agencies: [1, 2, 3, 4, 5, 6],
+};
```

Comment:
This is just a random filter to get some data into the page. Future PRs will pull the agency from the user info object (or wherever that's stored - can't remember now)